### PR TITLE
Various Templates

### PR DESCRIPTION
**Pull request by varunagrawal**
Fix for #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,29 @@
...
+Official Command Line utility to use EvalAI in your terminal. 
 
-## Setting up EvalAI-CLI
+This package offers almost all of the full suite of features available on the website within your terminal.
 
-Setting up Virtual Environment.
+## Development Setup
 
-```
+1. Clone the repository to you machine
```
Comment (RishabhJain2018) [inline README.md]: Please change `you` to `your` and I think we can also add the `git clone` command for the easiness of the contributor.
`git clone https://github.com/Cloud-CV/evalai-cli.git EvalAI-CLI`

**Comment by varunagrawal** [comment]
@RishabhJain2018 done

File: README.md [hunk 2]
```diff
@@ -1,22 +1,33 @@
...
 
-`$ . venv/bin/activate`
 
-Installing your package locally.
+2. Install the package locally to try it out.
 
-`pip install --editable . `
\ No newline at end of file
+```bash
+$ pip install --editable .
```
Comment (deshraj) [inline README.md]: Can we replace this with `pip install -e .`

Comment (varunagrawal) [reply README.md]: Done

**Pull request merged**